semester-two/mcsl209/03-linked-list/find-common-elements.c

```diff
@@ -135,16 +139,21 @@ int main() {
 
     printf( "Enter the no. of items in Linked List A: " );
     scanf( "%d", &sizeA );
+    firstNodeA = createLinkedList( sizeA, firstNodeA );
 
     printf( "Enter the no. of items in Linked List B: " );
-    scanf( "%d", &sizeA );
+    scanf( "%d", &sizeB );
+    firstNodeB = createLinkedList( sizeB, firstNodeB );
```

Comment:
@sumayyahc1 What's the benefit of moving this up?

Reply:
@imranhsayed It's to separate the data entered for the two linked lists. It looked like this before, which was a bit confusing because it's not clear for which list the data is being entered:
![image](https://user-images.githubusercontent.com/80468902/191533418-f7c4e1ac-eb6a-496d-b1a8-0139a9a040b3.png)

Reply:
Fair enough!